ui/plugins/controller.js

```diff
@@ -146,7 +152,11 @@ treeherder.controller('PluginCtrl', [
                     // the fourth result comes from the jobLogUrl artifact
                     // exclude the json log URLs
                     $scope.job_log_urls = _.reject(results[3], {name: 'mozlog_json'});
-
+                    $scope.perf_job_url = ('http://local.treeherder.mozilla.org/perf.html#/graphs?series=[' +
+                    $scope.repoName+ ',' + $scope.job.signature + ',1]&selected=[' +
+                    $scope.repoName + ',' + $scope.job.signature + ',' + $scope.job.result_set_id +
+                        ',' + $scope.job.id + ']');
+                    console.log($scope.job.signature)
```

Comment:
Hi Will, I found the $scope.job.signature wasn't in the signature list which been retrieved by job_id endpoint, but they do share a same job_id(I mean the signature list been searched depend on job_id and the job_is is come from the 'job' we passed in). Is that possible? Can a job's signature miss by signature list which been searched by its job id?
@wlach